paypal/merchant-sdk-php/CVE-2017-6099.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Cross-site scripting (XSS) vulnerability in Paypal-Merchant-SDK-PHP
+link:      https://github.com/paypal/merchant-sdk-php/tree/v3.9.1
+cve:       CVE-2017-6099
+branches:
+    2.x:
+        time:     ~
+        versions: ['>=2.0.0', '<=2.11.118']
+    master:
+        time:     2017-03-31 08:33:00
```

Comment:
should be `2017-03-29 16:13:00` according to paypal/merchant-sdk-php@6389e9d6fed5a76ad69b382be7432e0664cb3298